test/suites/edgexfoundry/proxy_test.go

```diff
@@ -47,7 +45,7 @@ func TestAddProxyUser(t *testing.T) {
 	stdout, _ := utils.Exec(t,
 		fmt.Sprintf("edgexfoundry.secrets-config proxy adduser --token-type jwt --user example --algorithm ES256 --public_key %s --id 1000 -jwt %s",
 			publicKey,
-			kongAdminJWT))
+			string(kongAdminJWT)))
```

Comment:
```suggestion
			kongAdminJWT))
```
Sprintf's "%s" performs the bytes to string conversion internally: https://go.dev/play/p/_L26-czTqXA